content/legacy/_pub_1999_12_virtual-presentations.md

```diff
@@ -29,7 +29,7 @@ We thought about RealAudio/RealVideo broadcasting to bring a presentation to a r
 
 So, rather than trying to solve this problem with too many layers of technology, we just routed around it and used speaker phones. After all, if we are targeting small, far-flung groups, it should be easier to find a location with a speaker phone and a slow net connection than it is to find a location with sufficient unused bandwidth on a T1 line for realtime audio/video. A good host site can also use three-way calling to bring in two remote sites instead of one. While this technique may be much more limiting than using RealAudio or other internet audio broadcast, it is also much more accessible.
 
-Now that we had the two-way communication problem solved, all that was left was synchronizing the audio presentation to the slides. Luckily, Kevin Lenzo had already solved this exact problem already with his two 'SlideShow' scripts (<http://www.cs.cmu.edu/~lenzo/SlideShow/>).
+Now that we had the two-way communication problem solved, all that was left was synchronizing the audio presentation to the slides. Luckily, Kevin Lenzo had already solved this exact problem already with his two 'SlideShow' scripts (<http://web.archive.org/web/20070911001911/http://www.cs.cmu.edu:80/~lenzo/SlideShow/>).
```

Comment:
Make this a markdown link

Reply:
[promoting-perl-community-articles.zip](https://github.com/tpf/perldotcom/files/5901347/promoting-perl-community-articles.zip)

Sorry, I just noticed this option.  Will this work?

Reply:
Don't attach zip files. Will what work? 

It's easier to make the change and then I can just look.